.github/workflows/update_package_properties.yml

```diff
@@ -27,7 +27,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Install dependencies
-        run: sudo apt-get update && apt install libcurl4-openssl-dev libssl-dev
+        run: sudo apt-get update && sudo apt-get install libcurl4-openssl-dev libssl-dev
```

Comment:
update_package_properties pipeline failed since sudo is missing. I needed to execute manually for this time